sentimental/src/main/webapp/style.css

```diff
@@ -203,4 +203,12 @@ button:hover {
 #sentimentResult li {
     color: var(--dark);
     font-family: HK Light;
-}
\ No newline at end of file
+}
+
+html, #wordCloud {
+      width: 100%;
+      height: 800px;
+      margin: 0;
```

Comment:
We would probably want to change this to be width: 100vw and remove the height constraint.

Reply:
I just tried this, but if I add width: 100vw and remove the height constraint the word cloud shows squashed